homeassistant/components/isy994/__init__.py

```diff
@@ -386,6 +389,12 @@ def _categorize_weather(hass: HomeAssistant, climate) -> None:
     hass.data[ISY994_WEATHER].extend(weather_nodes)
 
 
+def _add_x10_filters():
+    global NODE_FILTERS
```

Comment:
`global` is not allowed. A refactor may be needed.

Reply:
I gave it a go at a refactor to avoid the usage of `global`. Thanks for the feedback Martin!

Reply:
I don't believe this refactor was necessary. It seems to me that the only issue was the `global NODE_FILTERS` line, which could simply be removed. The `NODE_FILTERS` variable is defined up at the top, same as a ton of others, and is accessible to any functions here without needing to use `global`. You'll note that `NODE_FILTERS` is used in a whole bunch of functions.